pages/docs/workflows.md

```diff
@@ -527,3 +527,40 @@ run = 'deno --unstable run -A --no-check --import-map importmap.json deno-genera
 ```
 
 For an easier `chomp deno-generate` execution that will first ensure the import map is up to date in the task graph.
+
+## Vitejs
+
+ES modules are supported directly in all major [browsers](https://caniuse.com/es6-module) now and with [es-modu-shims](https://github.com/guybedford/es-module-shims) you can polyfill the rest of the missing features and ship ES modules with confidence.
```

Comment:
It's usually best to avoid terms like "you" and instead use passive voice when possible.

```suggestion
ES modules are supported directly in all major [browsers](https://caniuse.com/es6-module) and with [es-module-shims](https://github.com/guybedford/es-module-shims) import maps and other features can be performantly polyfilled when not supported to allow shipping ES modules with confidence.
```